lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
Sure custom exception is fine. PR?
On Thu, 16 Apr 2015 at 16:34 Vincent Bonmalais <notifications@github.com>
wrote:

> In lib/ably/realtime/channel.rb
> <https://github.com/ably/ably-ruby/pull/22#discussion_r28520580>:
>
> > @@ -179,15 +184,26 @@ def presence
> >          @presence
> >        end
> >
> > -      # Return the message history of the channel.
> > +      # Return the message history of the channel
> > +      #
> > +      # Once attached to a channel, you can retrieve messages published on the channel before the
>
> Maybe it would make sense to throw a custom exception in this case since
> it is dependent of the Channel state and that, in fact, the option is valid?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ably/ably-ruby/pull/22/files#r28520580>.
>